docs/user-guide/user-manual/configuration.md

```diff
@@ -19,17 +19,17 @@ note that in the configuration files the leading dash(es) are omitted (see
 
 ```{eval-rst}
 --max-memory=value, -m value
-              [Set the limit for the *Java Virtual Machine*’s heap
-              size](advanced-use.html#limit-the-maximum-java-heap-size-memory-use)
-              (`-Xmx`).  This option accepts either absolute values (integer or
+              `Set the limit for the Java Virtual Machine’s heap
+              size <advanced-use.html#limit-the-maximum-java-heap-size-memory-use>`_
+              (``-Xmx``).  This option accepts either absolute values (integer or
               decimal), optionally with a suffix to indicate Mibibytes,
-              Gibibytes, or Tebibytes: `M`, `G`, `T`), or relative, expressed in
-              a percentage of total memory, with a `%` suffix. Default: `80%`
+              Gibibytes, or Tebibytes: ``M``, ``G``, ``T``), or relative, expressed in
+              a percentage of total memory, with a ``%`` suffix. Default: ``80%``
 
 --r5-classpath=classpath, -r classpath
               Point to R⁵’s JAR (or build directory) in case you want to use a
-              [custom R⁵ installation](advanced-use.html#use-a-custom-installation-of-r5).
-              Default: `""` (download latest compatible version of R5)
+              `custom R⁵ installation <advanced-use.html#use-a-custom-installation-of-r5>`_ .
+              Default: ``""`` (download latest compatible version of R5)
```

Comment:
Because this is within an `{eval-rst}` block, everything has to be rst rather than md (myst-md do not yet support option-lists, but they are underway)